web/pdf_rendering_queue.js

```diff
@@ -102,15 +102,22 @@ class PDFRenderingQueue {
    * @param {Array} views
    * @param {boolean} scrolledDown
    * @param {boolean} [preRenderExtra]
+   * @param {boolean} [ignoreDetailViews]
    */
-  getHighestPriority(visible, views, scrolledDown, preRenderExtra = false) {
+  getHighestPriority(
+    visible,
+    views,
+    scrolledDown,
+    preRenderExtra = false,
+    ignoreDetailViews = false
```

Comment:
Possibly a pointless optimization, but would it make sense to always pass `ignoreDetailViews === true` when rendering the thumbnails since those by design won't have them?

https://github.com/mozilla/pdf.js/blob/d1f62509e531b9e9f8862e3526eaa17565ff1e9c/web/pdf_thumbnail_viewer.js#L289-L296

Reply:
I don't expect it to have a any significant effect, but doing it doesn't hurt for sure.